src/middleware/routeHandler.js

```diff
@@ -25,14 +25,13 @@ const gitFileSystemService = new GitFileSystemService(
   new SimpleGit({ maxConcurrentProcesses: MAX_CONCURRENT_GIT_PROCESSES })
 )
 
-const isRepoWhitelisted = (siteName, ggsWhitelistedRepos) => {
+const isRepoWhitelisted = (siteName, growthbook) => {
```

Comment:
do we still need a function for this? seems like the sitename is only passed in to log it and we are just calling 1 growthbook function

Reply:
removed